### ISSUE TITLE
Missing Windows Download

### ISSUE THREAD

**Issue by jocko96b** [complexlogic/big-launcher#11]
The link in the documentation to download the latest Windows build is broken (https://nightly.link/complexlogic/big-launcher/workflows/build/master/Windows%20build.zip)

**Comment by jocko96b** [comment]
Awesome - thanks!  I was in the middle of trying to get my build environment working on Windows after getting it to build in Ubuntu, but it was slow going (I am a software test by trade, not development lol).  

Awesome work on this - I haven't found anything out there that does what Big-Launcher (or Flex-Launcher) does and this is perfect for what I am trying to do.

**Comment by complexlogic** [comment]
I agree, very perplexed at how few solutions exist for the task of switching between a handful of applications using a remote/gamepad interface,
